drizzlepac/runastrodriz.py

```diff
@@ -813,12 +815,14 @@ def verify_alignment(inlist, calfiles, calfiles_flc, trlfile,
                     else:
                         trlstr = "Could not align {} to absolute astrometric frame\n"
                         trlmsg += trlstr.format(row['imageName'])
+                        print(trlmsg)
                         return None
             except Exception:
                 # Something went wrong with alignment to GAIA, so report this in
                 # trailer file
                 _trlmsg = "EXCEPTION encountered in align...\n"
                 _trlmsg += "   No correction to absolute astrometric frame applied!\n"
+                print(_trlmsg)
```

Comment:
Since the message is going to the Trailer file, do we need a print statement here?

Reply:
Unfortunately, these comments may be recorded in the trailer file, but the end user (or pipeline) does not know why the code quits early without sending these comments to stdout through this 'print()' statement.